core/tests/webdriverio_utils/ExplorationEditorPage.js

```diff
@@ -164,9 +178,13 @@ var ExplorationEditorPage = function() {
       explorationMetadataModalHeaderElement);
     await action.waitForAutosave();
 
-    await action.select(
-      'Exploration Language', expLanguageSelectorElement,
-      language);
+    await action.click(
+      'Exploration Language', expLanguageSelectorElementModal);
+
+    await action.click(
+      'Language input Choice takes too long to be visible. ',
+      languageChoiceOptionElement(language)[0]);
```

Comment:
It will return undefined here as we are not using await with `$$` selector. It should be like this (Also the first argument in action.click() is element name not error message):

```
var languageChoiceOption = await languageChoiceOptionElement(language)[0];
await action.click(
      'Language Input Choice ', languageChoiceOption);
```

Reply:
done

Reply:
Not done '(Also the first argument in action.click() is element name not error message):'